src/purifier.js

```diff
@@ -390,17 +390,22 @@ export function validateAttributeChange(purifier, node, attr, value) {
   if (whitelist) {
     const {attribute, values} = whitelist;
     if (attribute === attr) {
-      if (value == null || !values.includes(value)) {
+      if (value === null || !values.includes(value)) {
```

Comment:
Do we know the value can't be `undefined`?

Reply:
Yea, the TS interface uses `string | null`. Maybe it's better to be defensive than readable here though.